tools/bench-libs/src/main/java/org/enso/benchmarks/libs/LibBenchRunner.java

```diff
@@ -60,7 +66,37 @@ private void collectBenchSpecs() {
             "enso.languageHomeOverride",
             Paths.get("../../distribution/component").toFile().getAbsolutePath()
         ).build();
+  }
 
-    ctx.eval()...
+  private static void closeCtx() {
+    ctx.close();
+  }
+
+  public void run(String label) {
+    ChainedOptionsBuilder builder = new OptionsBuilder()
+      .jvmArgsAppend("-Xss16M", "-Dpolyglot.engine.MultiTier=false")
+      .include("^" + label + "$");
+  }
+
+  private static Set<BenchSuite> collectAllBenchSpecs() {
+    String benchmarksProjectRootDir = "test/Benchmarks/src";
+    File singleFile = new File("/home/pavel/dev/enso/engine/runtime/src/bench/resources/org.enso.interpreter.bench.benchmarks.meso/simple.enso");
```

Comment:
Let's try to locate the benchmarks via relative way. For example like https://github.com/enso-org/enso/pull/7101/files/f6dd4237d5ab3ca3011c1c4ae60358ba8f4c5b4e..9d9f328ec384e303d4cdf6a23e6deccd2cf8c755#diff-a8c733e77d2301c4075061f5f53e29e7aea47170134995974796c14ba20231c7R40